gcovr/tests/html-css/Makefile

```
@@ -0,0 +1,17 @@
+all:
+	$(CXX) -fprofile-arcs -ftest-coverage -fPIC main.cpp -o testcase
+
+run: html css
+
+html:
+	./testcase
+	$(GCOVR) -d --html-details --html-css=style.css -o coverage.html
```

Test can be minimized by avoiding --html-details, or by setting --html-self-contained
```suggestion
	$(GCOVR) -d --html --html-css=style.css -o coverage.html
```

---

Ok.